### DISCUSSION TITLE
.numbered for numbering select sections?

### DISCUSSION THREAD

**Discussion by dhicks** [quarto-dev/quarto-cli#9578] [Q&A]
### Description

I'm preparing a journal article for submission.  The journal doesn't use numbered sections, but I'd like to have numbered sections in the supplemental info, to facilitate cross-referencing from the main body to the SI.  I know that I can set `number-sections: true` and then add `{.unnumbered}` to each of the main body sections.  But there are a lot more main body sections than SI sections, so it would be convenient to be able to go the other direction: set `number-sections: false` and then add `{.numbered}` to the supplement sections.  But this doesn't appear to work?

**Comment by cscheid** [comment]
I think that's right, that symmetry doesn't exist today (although it should!).

Inspecting at our code, we process `unnumbered` classes in `quarto_crossref_filters`, which happen after user filters. That means that it should be possible for you to write Lua filters that implement this symmetry yourself.
